src/Symfony/Component/Form/FormEvents.php

```diff
@@ -12,7 +12,7 @@
 
 /**
  * To learn more about how form events work check the documentation
- * entry at {@link http://symfony.com/doc/any/components/form/form_events.html}
+ * entry at {@link http://symfony.com/doc/any/components/form/form_events.html}.
```

Comment:
I might be wrong, but I think (from what I remember the last time I read the phpdoc "specs") the short description ends if there are 2 empty lines or at the first dot.